BRANDING.md

```diff
@@ -0,0 +1,55 @@
+# Istio Brand Guidelines
+ 
+The Istio trademark and logos (“Istio Trademarks”) are treated separately from the copyright or patent license grants contained in the Apache-licensed Istio repositories on GitHub. Any use of the Istio Trademarks other than those permitted in these guidelines must be approved in advance.
+
+## Purpose of the Brand Guidelines
+    
+These guidelines exist to ensure that the Istio project can share its platform under open source licenses while making sure that the “Istio” brand is protected as a meaningful source identifier in a way that is consistent with trademark law.
+ 
+## Acceptable Uses
+
+Given the open nature of Istio, you may use the Istio trademark to refer to the project without prior written permission. Examples of these acceptable references include the following:
+
+-   To refer to the Istio project itself;
+    
+-   To refer to unmodified source code or other files shared by the Istio repositories on GitHub;
+    
+-   To accurately identify that your design or implementation is based on or is for use with the Istio platform.
```

Comment:
for consistency you should end with ";" here and then add a "." to the next one. Or just remove them from all lines